programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -1634,6 +1634,34 @@ static void run(int argc, char *argv[]) {
     signal(SIGINT, interrupt_handler);
 #endif
 
+#if _WIN32
+    if (init) {
+        ziti_log_init(ziti_loop, ZITI_LOG_DEFAULT_LEVEL, ziti_log_writer);
+        struct tm *start_time = get_log_start_time();
+        char time_val[32];
+        strftime(time_val, sizeof(time_val), "%a %b %d %Y, %X %p", start_time);
+        ZITI_LOG(INFO,"============================ service begins ================================");
+        ZITI_LOG(INFO,"Logger initialization");
+        ZITI_LOG(INFO,"	- initialized at   : %s", time_val);
+        ZITI_LOG(INFO,"	- log file location: %s", get_log_file_name());
+        ZITI_LOG(INFO,"============================================================================");
+    } else {
+        ziti_log_init(ziti_loop, ZITI_LOG_DEFAULT_LEVEL, NULL);
+    }
+    move_config_from_previous_windows_backup(ziti_loop);
+#else
+    ziti_log_init(ziti_loop, ZITI_LOG_DEFAULT_LEVEL, NULL);
+#endif
+
+    // set log level from instance/config, if NULL is returned, the default log level will be used
+    const char* log_lvl = get_log_level();
+    if (log_lvl != NULL) {
+        ziti_log_set_level_by_label(log_lvl);
+    }
+    ziti_tunnel_set_log_level(ziti_log_level());
+    set_log_level(ziti_log_level_label());
```

Comment:
`set_log_level((char *)ziti_log_level_label());`

clang produces a warning without the explicit cast from `const char *` to `char *`:

`passing 'const char *' to parameter of type 'char *' discards qualifiers [clang-diagnostic-incompatible-pointer-types-discards-quali    fiers]`